apps/dashboard/src/error-analysis/__mock_data__/dummyTreeAdultCensusIncome.ts

```diff
@@ -26,7 +26,7 @@ export const dummyTreeAdultCensusIncomeData: IErrorAnalysisTreeNode[] = [
   {
     arg: [1],
     badFeaturesRowCount: 0,
-    condition: "marital-status !=  Married-civ-spouse",
+    condition: "marital-status not equal to Married-civ-spouse",
```

Comment:
should this all be localized?

Reply:
Good point, looks like we have quite a few places where we don't localize the strings, not only this one. Let me update this one for now and I have reached out to PM and Ke for other localizations